css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+  font-family: "DM sans", sans-serif;
+  margin: 0;
+}
+
+p {
+  font-size: 0.5rem;
+  margin-top: 3px;
+}
+
+h4 {
+  text-align: center;
+  margin: 10px 0;
+}
+
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.wrap {
+  flex-wrap: wrap;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-fifth {
+  flex-basis: 20%;
+}
+
+.img-size {
+  width: 125px;
+  height: 175px;
+}
+
```

Comment:
You should use `object-fit` so that the image's aspect ratio doesn't get skewed by setting both a width and a height.